slides/08-Platform-Hub-New/1050-ProjectsAndKubeTokens.md

```diff
@@ -0,0 +1,13 @@
+## Projects and Kube Tokens
+
+ - Projects in which you are a team member of can be found in the Projects tab within the Org section
+ - You should now be able to see Developer-Induction project in the list
```

Comment:
Don't need a `-` in `Developer-Induction` – we should call the project `Dev Induction`, or something like that.